office365_admin/plugin.spec.yaml

```diff
@@ -4,7 +4,7 @@ products: [insightconnect]
 name: office365_admin
 title: Office365 Admin
 description: Administrate and manage Office 365 users and licenses
-version: 1.3.1
+version: 1.3.0
```

Comment:
Version should not downgrade, it should be 1.4.0

Reply:
I am not sure where this diff is coming from, but there was 1 intermediate commit I had where I took the version from 1.2.1 to 1.3.1.  That was too much.  Before I started working on this, the plugin version was 1.2.1.  It should be 1.3.0 after my work.  Sorry for the confusion here.

Reply:
1.3.1 has already been released - therefore we cannot go back in versions. The version will have to increment.